src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -340,13 +353,34 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       parsedMessageContent += token;
     }
 
-    Message message =
-        new Message(
-            UUID.randomUUID(),
-            conversation.getId(),
-            user.getId(),
-            parsedMessageContent,
-            Instant.now());
+    Message message;
+
+    Part filePart = request.getPart("image"); // Retrieves <input type="file" name="file">
+    if(filePart != null && !filePart.getSubmittedFileName().equals("")){
+      InputStream fileContent = filePart.getInputStream();
+
+      // ImagesService imagesService = ImagesServiceFactory.getImagesService();
```

Comment:
nit: Avoid checking in commented code (since Git will track stuff over time anyway).